docs/websocket/audio.md

```diff
@@ -20,3 +20,17 @@ name         | type  | description
 channels     | `int` | **Required** The number of channels.
 frame_rate   | `int` | **Required** The frame rate in Hertz.
 sample_width | `int` | **Required** The sample width in bits.
+
+
+## Self-describing formats
+
+A self-describing audio format has a file header with all required information.
+The [SoX documentation][1] elaborates on this in more technical details.
+
+Basically, any self-describing audio format *should* work, however only WAVE is
+supported.
+
+To support streams, WAVE for example omits the length of the audio in the
```

Comment:
perhaps use a heading: WAVE to explain the details about how we support wave as official format as well as a hack without the length. Perhaps a link to this 'extended standard'

Reply:
I'll just omit this, I might be wrong here. After reading something about it (http://soundfile.sapp.org/doc/WaveFormat/) it seems that omitting the size in the header means each chunk should have a size 'header'. I think I'm adding more confusion than clarity here.